daemon/config.go

```diff
@@ -206,6 +212,34 @@ func getConflictFreeConfiguration(configFile string, flags *flag.FlagSet) (*Conf
 			return nil, err
 		}
 
+		// make sure that values set in the config file with nullable values, like `false`,
+		// are not overriden by default truthy values from the flags that were not explicitly set.
```

Comment:
Infact, this entire new code can be moved down to findConfigurationConflicts and merged with L 272-278.